Makefile

```diff
@@ -4,48 +4,44 @@
 
 GOBIN = $(shell pwd)/build/bin
 GOFMT = gofmt
-GO ?= latest
+GO ?= 1.13.1
```

Comment:
It seems a bit confusing with dockerfile using 1.12 or 1.11 of go, but the make file using 1.13?
Shall we fix a GO version(1.13) across the repo?